backend/src/packages/auth/auth.service.ts

```diff
@@ -1,25 +1,51 @@
+import { type Token } from '~/libs/packages/token/token.js';
 import {
+  type UserService,
   type UserSignInRequestDto,
+  type UserSignInResponseDto,
   type UserSignUpRequestDto,
   type UserSignUpResponseDto,
-} from '~/packages/users/libs/types/types.js';
-import { type UserService } from '~/packages/users/user.service.js';
+} from '~/packages/users/users.js';
+
+import { ExceptionMessages } from './libs/enums/enums.js';
+import { ApplicationError } from './libs/exceptions/exceptions.js';
 
 class AuthService {
   private userService: UserService;
-
-  public constructor(userService: UserService) {
+  private tokenService: Token;
+  public constructor(userService: UserService, token: Token) {
     this.userService = userService;
+    this.tokenService = token;
+  }
+
+  public async signIn(
+    userRequestDto: UserSignInRequestDto,
+  ): Promise<UserSignInResponseDto | null> {
+    const { email } = userRequestDto;
+    return await this.login(email);
   }
 
-  public signIn(userRequestDto: UserSignInRequestDto): UserSignInRequestDto {
-    return userRequestDto;
+  private async login(email: string): Promise<UserSignInResponseDto> {
+    const user = await this.userService.findByEmail(email);
+    if (!user) {
+      throw new ApplicationError({
+        message: ExceptionMessages.userNotFound,
```

Comment:
```suggestion
        message: ,
```
do not need an enum for now